coalib/bears/meta.py

```diff
@@ -22,4 +24,7 @@ def __init__(cls, clsname, bases, clsattrs, *varargs, aspects=None):
         """
         type.__init__(cls, clsname, bases, clsattrs, *varargs)
         if aspects is not None:
-            cls.aspects = defaultdict(lambda: [], aspects)
+            for k, v in aspects.items():
+                aspects[k] = aspectlist(v)
```

Comment:
This is not a good idea, since it modifies the original `aspects` object given by the user to the `class` definition, which furthermore doesn't necessarily need to be a `dict` object. Any mapping accepted by `dict.__init__` should be supported there. So better do something like:

```python
cls.aspects = defaultdict(
    lambda: aspectlist([]),
    ((k, aspectlist(v) for k, v in dict(aspects).items()))
```